public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/appInsightsGeoMap.json

```diff
@@ -822,7 +822,7 @@
         "options": [],
         "query": {
           "grafanaTemplateVariableFn": {
-            "kind": "MetricDefinitionsQuery",
+            "kind": "MetricNamespaceQuery",
```

Comment:
usually it's good to leave these queries as the "old" way to check if we break migrations but for this case I guess it's okay to update them.

Reply:
I guess it did not help us spot that we did break migrations ;)

I would rather update them in case someone uses them as template to create another curated dashboard. 
But we could leave one (not importable) and use it in the tests...
WDYT?

Reply:
yep, it's not automated so it's difficult to catch errors ... but it's helpful to reproduce issues :) anyway I also get your point for people using these as templates so it's also fine to update them.